src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.IO;
```

Comment:
We don't have a direct dependency to `System.IO`, instead we use an abstraction `FileSystemProvider`.